### PR TITLE
[UI] fixes some minor UI issues

### PR DESCRIPTION
**Pull request by maxlang**
- added favicon
- tweaked table byte styles
- fixed table horizontal scrolling
- page title -> "Cockroach Console"
- chart number format fixes
- remove google font

[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/3598)

### CODE REVIEW

**Comment by tamird** [comment]
Can you make the commit message match the PR description?

**Comment by tamird** [comment]
This favicon is a gigantic 256x256, is that really necessary?

File: ui/debug/index.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
     <script src="/bower_components/nvd3/build/nv.d3.js"></script>
     <script src="/bower_components/moment/moment.js"></script>
     <script src="/bower_components/moment-timezone/builds/moment-timezone-with-data.js"></script>
-    <title>Cockroach</title>
+    <title>Cockroach Console</title>
```
Comment (tamird) [inline ui/debug/index.html]: while you're here, "CockroachDB"?

**Comment by maxlang** [comment]
- Fixed commit message.
- I just used the same favicon from the blog, but I can probably generate a smaller one, which we should probably use for the blog too. Any tools you'd recommend for this?
- I feel "Cockroach Console" is cleaner than "CockroachDB Console" but it's up for debate

**Comment by tamird** [comment]
> I just used the same favicon from the blog, but I can probably generate a smaller one, which we should probably use for the blog too. Any tools you'd recommend for this?

nope, dunno. why don't we have an svg of the logo?

**Comment by maxlang** [comment]
We do, but favicon svgs aren't really supported http://caniuse.com/#feat=link-icon-svg

**Comment by maxlang** [comment]
Apparently favicon stuff is complicated https://css-tricks.com/favicon-quiz/

I just copied the blog for now.

**Comment by maxlang** [comment]
As another small change, I removed the google font we were using (and added that to the commit message)

**Comment by tamird** [comment]
can you make the commit message summary (the first line) more descriptive? "minor fixes" isn't very informative. I'd have put these changes in separate commits, but I won't insist on that.

**Comment by maxlang** [comment]
fixed commit message

File: ui/Makefile [hunk 2]
```diff
@@ -65,7 +67,9 @@ GOBINDATA_DEBUG_SOURCES := $(BOWER_COMPONENTS)/d3/d3.js \
                            $(BOWER_COMPONENTS)/nvd3/build/nv.d3.js \
                            $(BOWER_COMPONENTS)/moment/moment.js \
                            $(BOWER_COMPONENTS)/moment-timezone/builds/moment-timezone-with-data.js \
-                           $(GOBINDATA_FONTS)
+                           $(GOBINDATA_FONTS) \
+                           favicon.ico \
+                           apple-touch-icon.png
```
Comment (tamird) [inline ui/Makefile]: can we name this file `favicon.ico`? it seems more appropriately sized.

Comment (maxlang) [inline ui/Makefile]: favicon.ico is actually an ico file with 4 different sizes in it - 16x16, 32x32, 48x48 and, 256x256

The higher resolutions are for things like icons (eg how it appears in the windows taskbar) and 256x256 covers most of the edge cases, which is probably why it's included, but could probably be removed without affecting anything.

That said, favicons are cached very aggressively so it won't incur any cost after the first load and hedges our bets so we'll look good on any system where the icon appears.

I'm not a favicon expert, but this seems to be the idea behind the images from the blog.

Comment (tamird) [inline ui/Makefile]: I see.

**Comment by maxlang** [comment]
screen shot 2016-01-05 at 1 01 43 pm
screen shot 2016-01-05 at 1 01 39 pm
screen shot 2016-01-05 at 1 01 11 pm
screen shot 2016-01-05 at 1 01 07 pm
screen shot 2016-01-05 at 12 56 24 pm

**Comment by tamird** [comment]
LGTM but I didn't review the css and typescript changes; @mrtracy should given final approval here.

**Comment by maxlang** [comment]
Thanks, sounds good.

**Comment by mrtracy** [comment]
LGTM

---

Review status: 0 of 15 files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/3598#-:-K7It4eCCy7vZmJNfTS1:676834233)*

**Comment by mrtracy** [comment]
Reviewed 9 of 13 files at r1, 6 of 6 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/3598)*

**Pull request merged**